dev.Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM --platform="linux/arm64" arm64v8/node@sha256:56e8282f4392fb96c877babc93b3829e46b79c6fbcd48c92de578febffc80587 AS builder
+ARG NODE_SHA=sha256:56e8282f4392fb96c877babc93b3829e46b79c6fbcd48c92de578febffc80587
+ARG TARGET_PLATFORM=linux/arm64
+
+FROM --platform=${TARGET_PLATFORM} arm64v8/node@${NODE_SHA} AS builder
```

Comment:
Does this work?
I assumed the SHA would point to a platform-specific version of the base image but I haven't verified this.

Also the image ID is prefixed `arm64v8` so doesn't that lock in the platform?